modules/administration/pages/hostname-rename.adoc

```diff
@@ -0,0 +1,17 @@
+[[hostname-rename]]
+= Rename Hostname
+
+With the [command]``spacewalk-hostname-rename`` script it is possible to rename the hostname of the {productname} Server.
+The [command]``spacewalk-hostname-rename`` script is part of the [package]``spacewalk-utils`` package.
+
+The only mandatory parameter for the script is the newly configured IP address of the {productname} Server.
+It is the best to proceed as follows: 
```

Comment:
Make it mandatory. I don't know what happens if the user do it otherwise, but maybe things will not work :-)